package.json

```diff
@@ -47,10 +52,12 @@
     "ws": "2.3.1"
   },
   "scripts": {
-    "build": "babel src --out-dir .",
+    "build": "babel src --out-dir . && flow-copy-source src .",
     "webpack": "webpack",
-    "dev": "babel src --out-dir ${WEBAPP_DIR:-../mattermost-webapp}/node_modules/mattermost-redux --source-maps",
-    "dev-mobile": "babel src --out-dir ${MOBILE_DIR:-../mattermost-mobile}/node_modules/mattermost-redux --source-maps",
+    "dev": "babel src --out-dir ${WEBAPP_DIR:-../mattermost-webapp}/node_modules/mattermost-redux --source-maps && yarn dev:flow-copy",
+    "dev:flow-copy": "flow-copy-source src ${WEBAPP_DIR:-../mattermost-webapp}/node_modules/mattermost-redux",
+    "dev-mobile": "babel src --out-dir ${MOBILE_DIR:-../mattermost-mobile}/node_modules/mattermost-redux --source-maps && yarn dev-mobile:flow-copy",
+    "dev-mobile:flow-copy": "flow-copy-source src ${MOBILE_DIR:-../mattermost-webapp}/node_modules/mattermost-redux",
```

Comment:
What's the purpose of the `:flow-copy` commands? Non-blocking comment, just curious

Reply:
the purpose is same with `.d.ts` of TypeScript.

flow type annotations will be removed after transpile. so it makes sense copying sources to another directory before transpile to use types in other projects which uses mattermost-redux.